server/zanata-security/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.zanata</groupId>
+    <artifactId>server</artifactId>
+    <version>4.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>zanata-security</artifactId>
+  <name>zanata-security</name>
```

Comment:
In terms of naming this `zanata-security`: right now this module just has the interfaces Identity, Authenticated and AuthenticatedLiteral. Is it likely to get most of the security code in future? It's not going to have three classes forever, so what are they going to have in common? I'm not sure zanata-authentication works either, given that our actual authentication code will probably be staying elsewhere.

I wonder if Identity should have a different name. That might inform the decision on the module name.